prime-angular-frontend/src/app/modules/paper-enrolment/pages/self-declaration-page/self-declaration-overview.component.ts

```diff
@@ -10,6 +10,7 @@ import { SelfDeclarationForm } from './self-declaration-form.model';
   selector: 'app-self-declaration-overview',
   template: `
     <app-enrollee-self-declarations *ngIf="selfDeclarations?.selfDeclarations"
+                                    [showRedirect]="true"
```

Comment:
I notice that some of the sections for Paper Enrollment overview have the new Edit button whereas some don't 
![image](https://user-images.githubusercontent.com/7586665/187830548-9d5dfcff-d8c4-446b-a8ca-78615edf084e.png)

The Edit button is more important for the external user so can it be consistent, either no Edit button for all sections or yes.  Question for client?